tests/test_indexing.py

```
@@ -646,6 +768,32 @@ def test_jsonl_supplements_sessions_index_work_days(self, env):
         assert "2026-02-05" in data["workDays"]  # from JSONL
         assert "2026-02-10" in data["workDays"]  # from JSONL
 
+    def test_transcript_cwd_wins_over_stale_index_path(self, env):
+        """When both exist, the path comes from the transcript cwd, not the
+        (possibly stale) sessions-index.json originalPath; index work days are
+        still unioned in."""
+        projects_dir, memory_dir, index_file = env
```

Ruff (0.15.18)

[warning] Unpacked variable `memory_dir` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)